x-pack/plugins/security_solution/public/common/components/event_details/insights/simple_alert_table.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import type { EuiBasicTableColumn } from '@elastic/eui';
+import type { Severity } from '@kbn/securitysolution-io-ts-alerting-types';
+import { EuiBasicTable, EuiLoadingContent, EuiSpacer } from '@elastic/eui';
+
+import { PreferenceFormattedDate } from '../../formatted_date';
+import { SeverityBadge } from '../../../../detections/components/rules/severity_badge';
+import { useAlertsByIds } from '../../../containers/alerts/use_alerts_by_ids';
+import { SIMPLE_ALERT_TABLE_ERROR, SIMPLE_ALERT_TABLE_LIMITED } from './translations';
+
+const TABLE_FIELDS = ['@timestamp', 'kibana.alert.rule.name', 'kibana.alert.severity'];
+
+const columns: Array<EuiBasicTableColumn<Record<string, string[]>>> = [
+  {
+    field: 'kibana.alert.rule.name',
+    name: 'Rule',
+  },
+  {
+    field: '@timestamp',
+    name: '@timestamp',
+    render: (timestamp: string) => <PreferenceFormattedDate value={new Date(timestamp)} />,
+  },
+  {
+    field: 'kibana.alert.severity',
+    name: 'Severity',
+    render: (severity: Severity) => <SeverityBadge value={severity} />,
+  },
+];
+
+const alertLimit = 10;
```

Comment:
Can you add a note why it's limited to 10 in case there's a desire to change it in the future?

Reply:
Sure thing: https://github.com/elastic/kibana/pull/136009/commits/0c3cdcdd99df85fedb7aa71b73019a03b536a9c0